### PR TITLE
Sit 

### PR DESCRIPTION
**Pull request by RebeccaStankus**
https://highfidelity.atlassian.net/browse/BUGZ-655

### CODE REVIEW

File: usefulUtilities/sitV2/appResources/appData/sitClientV2.js [hunk 1]
```diff
@@ -585,7 +583,7 @@
 
     // Can sit when clicking on chair when enabled via userData
     function mousePressOnEntity(id, event) {
-        if (event.isPrimaryButton && !isInEditMode()) {
+        if (event.isPrimaryButton && !Settings.getValue(EDIT_SETTING)()) {
```
Comment (zfox23) [inline usefulUtilities/sitV2/appResources/appData/sitClientV2.js]:
```suggestion
        if (event.isPrimaryButton && !Settings.getValue(EDIT_SETTING, false)) {
```

File: usefulUtilities/sitV2/appResources/appData/sitClientV2.js [hunk 2]
```diff
@@ -353,12 +350,13 @@
...
     // Remotely called from canSitZone
     var AVATAR_SITTING_IN_CHAIR_RANGE = 0.01;
+    var EDIT_SETTING = "io.highfidelity.isEditing";
     function onEnterCanSitZone(id, params) {
         if (params && params.length > 0) {
             _this.zoneID = params[0];
         }
 
-        if (!isInEditMode()) {
+        if (!Settings.getValue(EDIT_SETTING)) {
```
Comment (zfox23) [inline usefulUtilities/sitV2/appResources/appData/sitClientV2.js]:
```suggestion
        if (!Settings.getValue(EDIT_SETTING), false) {
```

File: usefulUtilities/sitV2/appResources/appData/sitClientV2.js (hunk 1 above)
Comment (RebeccaStankus) [reply usefulUtilities/sitV2/appResources/appData/sitClientV2.js]: ack

File: usefulUtilities/sitV2/appResources/appData/sitClientV2.js (hunk 2 above)
Comment (RebeccaStankus) [reply usefulUtilities/sitV2/appResources/appData/sitClientV2.js]: ack

File: usefulUtilities/sitV2/appResources/appData/sitClientV2.js [hunk 3]
```diff
@@ -585,7 +583,7 @@
 
     // Can sit when clicking on chair when enabled via userData
     function mousePressOnEntity(id, event) {
-        if (event.isPrimaryButton && !isInEditMode()) {
+        if (event.isPrimaryButton && !Settings.getValue(EDIT_SETTING, false)()) {
```
Comment (zfox23) [inline usefulUtilities/sitV2/appResources/appData/sitClientV2.js]:
```suggestion
        if (event.isPrimaryButton && !Settings.getValue(EDIT_SETTING, false)) {
```

**Pull request merged**